Skip the "credential_helper5" test if git is not installled.
Index: git2/src/cred.rs
===================================================================
--- git2.orig/src/cred.rs
+++ git2/src/cred.rs
@@ -563,6 +563,9 @@ echo username=c
 
     #[test]
     fn credential_helper5() {
+        if !Path::new("/usr/bin/git").exists() {
+            return;
+        } //this test does not work if git is not installed
         if cfg!(windows) {
             return;
         } // shell scripts don't work on Windows
